1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -20,6 +64,7 @@ function playAlarm() {
 
 function pauseAlarm() {
   audio.pause();
+  flashInterval();
```

Comment:
This works well in stopping the alarm. If you click it before the alarm goes off, it has some unexpected behaviour. However, adding functionality to pause the countdown is an optional extra, so it's OK that you haven't added this - just be aware of the unexpected behaviour.